app/assets/javascripts/components/authentication/ShowUser.jsx

```diff
@@ -0,0 +1,105 @@
+class ShowUser extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {
+      error: '',
+      email: '',
+      first_name: '',
+      last_name: '',
+      avatar: '',
+      role: '',
+      id: '',
+    };
+  }
+
+  componentDidMount() {
```

Comment:
you can just set the initial states in the constructor instead of in `componentDidMount()`